software/postiz.yml

```diff
@@ -0,0 +1,12 @@
+name: Postiz
+website_url: https://postiz.com
+description: Self-hosted social media management software. It helps you to schedule posts, track the performance of your content, and manage all your social media accounts in one place (Alternative to Buffer, Hootsuite, Sprout Social).
```

Comment:
```suggestion
description: Shedule posts, track the performance of your content, and manage all your social media accounts in one place (Alternative to Buffer, Hootsuite, Sprout Social).
```

Reply:
Spelling mistake 🙈
"Schedule"

Reply:
I'll fix it then we can commit, good catch!